xdem/terrain.py

```diff
@@ -1012,7 +1012,7 @@ def get_terrain_attribute(
     output_attributes = [terrain_attributes[key].reshape(dem.shape) for key in attribute]
 
     if isinstance(dem, gu.Raster):
-        output_attributes = [gu.Raster.from_array(attr, transform=dem.transform, crs=dem.crs, nodata=None) for attr in output_attributes]
+        output_attributes = [gu.Raster.from_array(attr, transform=dem.transform, crs=dem.crs, nodata=-99999) for attr in output_attributes]
```

Comment:
It would probably be better to use the default for the float32 or float64 dtype (which I think is -99999, but it would be more transparent!)

Reply:
It complains when I do that because `_default_ndv` is not supposed to be used outside of GeoUtils "access to a protected member of a module".
And, actually, we don't define the types of outputs properly in `terrain.py` yet (some like hillshade are `uint8`). We could improve this once we specify the output type.

Reply:
Opened #302